src/games/strategy/common/ui/BasicGameMenuBar.java

```diff
@@ -172,10 +171,8 @@ protected void addGameNotesMenu(final JMenu parentMenu) {
     if (notesProperty != null && notesProperty.trim().length() != 0) {
       final String notes = LocalizeHTML.localizeImgLinksInHTML(notesProperty.trim());
       m_gameNotesPane = new SoftJEditorPane(notes);
-      parentMenu.add(SwingAction.of("Game Notes...", e -> {
-        SwingUtilities.invokeLater(new Runnable() {
-          @Override
-          public void run() {
+      parentMenu.add(SwingAction.of("Game Notes...", e ->
+          SwingUtilities.invokeLater(() -> {
```

Comment:
Right. It was already difficult to follow before any of the lambda refactor and now is probably even harder to follow. If you take a step back and look at addGameNotesMenu method its pretty brutal. In particular these couple of lines chained together are very difficult to determine what's going on:
```
      parentMenu.add(SwingAction.of("Game Notes...", e ->
          SwingUtilities.invokeLater(() -> {
            final JEditorPane pane = m_gameNotesPane.getComponent();
```